api/src/services/cache/mem-cache.ts

```diff
@@ -0,0 +1,93 @@
+import { cloneDeep } from 'lodash';
+import { CacheService } from './cache';
+import LRU from 'lru-cache';
+
+export class MemCache extends CacheService {
```

Comment:
(little question that came up after peeking at this PR due to #17562)

Understandably this is backport from the original `drop-keyv` branch, but thoughts on naming this class as `MemoryCache` instead of `MemCache`? Technically the popular Memcached ends with an additional `d` so it's not really the same, but wondering would `MemoryCache` that is slightly more explicit be better for the long run.

Reply:
Fully agree :)